lib/mayaUsd/fileio/jobs/jobArgs.cpp

```diff
@@ -250,6 +250,13 @@ UsdMayaJobExportArgs::UsdMayaJobExportArgs(
                     UsdGeomTokens->bilinear,
                     UsdGeomTokens->none
                 })),
+        defaultUSDFormat(
+            _Token(userArgs,
+                UsdMayaJobExportArgsTokens->defaultUSDFormat,
+                UsdMayaTranslatorTokens->UsdFileExtensionCrate,
+                {
+                    UsdMayaTranslatorTokens->UsdFileExtensionASCII
+                })),                 
```

Comment:
Since we're talking about file formats here, I think it might be slightly more correct to use the format ID tokens provided by USD:

```C++
#include <pxr/usd/usd/usdaFileFormat.h>
#include <pxr/usd/usd/usdcFileFormat.h>
...
                UsdUsdcFileFormatTokens->Id,
                {
                    UsdUsdaFileFormatTokens->Id
                })),
...
```